README.md

```diff
@@ -4,7 +4,7 @@
   [![Stability](https://img.shields.io/badge/Stability-alpha-f4d03f.svg)](https://github.com/Qiskit-Extensions/circuit-knitting-toolbox/releases)
   ![Platform](https://img.shields.io/badge/Platform-Linux%20%7C%20macOS-informational)
   [![Python](https://img.shields.io/badge/Python-3.8%20%7C%203.9%20%7C%203.10%20%7C%203.11-informational)](https://www.python.org/)
-  [![Qiskit](https://img.shields.io/badge/Qiskit-%E2%89%A5%200.42.1-6133BD)](https://github.com/Qiskit/qiskit)
+  [![Qiskit](https://img.shields.io/badge/Qiskit-%E2%89%A5%200.43.0-6133BD)](https://github.com/Qiskit/qiskit)
```

Comment:
Allows us to more easily inspect and handle the `options` field of a `Sampler`. This will continue making our life easier as we develop around the Sampler, we should just bump it now

Reply:
The only downside I see with bumping it now is that, then, there is no Qiskit version supported that will run without deprecation warnings (#161).  But, since this is alpha software, I think this is okay.

Reply:
I am going to do the best I can with those warnings for 0.3.0. I believe updating EntForging to latest Qiskit and Qiskit Nature will resolve 95%+ of these